operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -19,7 +19,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
     // Prefix the dashboard title with "<product> /" unless configured otherwise.
     super.dashboard(
       title='%(prefix)s%(title)s' % { prefix: $._config.dashboard_prefix, title: title },
-      datasource=$._config.dashboard_datasource
+      datasource=$._config.dashboard_datasource, datasource_regex=$._config.datasource_regex
```

Comment:
I think I'm missing something: where is the code using `datasource_regex` when building the dashboard?

Reply:
The code using `datasource_regex` relies on the linked PR where the new parameter is added to [grafana-builder/grafana.libsonnet](https://github.com/grafana/jsonnet-libs/blob/master/grafana-builder/grafana.libsonnet#L2).

The resulting template looks  like that:
```diff
diff --git a/operations/mimir-mixin/config.libsonnet b/operations/mimir-mixin/config.libsonnet
index 345dd59a3..ee5db25d4 100644
--- a/operations/mimir-mixin/config.libsonnet
+++ b/operations/mimir-mixin/config.libsonnet
@@ -89,6 +89,6 @@

     // The default datasource used for dashboards.
     dashboard_datasource: 'default',
-    datasource_regex: '',
+    datasource_regex: 'demo-regex.*',
   },
 }
diff --git a/operations/mimir-mixin-compiled/dashboards/mimir-alertmanager-resources.json b/operations/mimir-mixin-compiled/dashboards/mimir-alertmanager-resources.json
index 10db97f71..3b2a09668 100644
--- a/operations/mimir-mixin-compiled/dashboards/mimir-alertmanager-resources.json
+++ b/operations/mimir-mixin-compiled/dashboards/mimir-alertmanager-resources.json
@@ -1263,7 +1263,7 @@
                "options": [ ],
                "query": "prometheus",
                "refresh": 1,
-               "regex": "",
+               "regex": "demo-regex.*",
                "type": "datasource"
             },
             {
```

Reply:
👍  Thanks for the info. Could you revert back the default regex to empty string, please?